internal/web_package/web_package.bzl

```diff
@@ -1,8 +1,38 @@
-def html_asset_inject(index_html, action_factory, injector, rootDirs, assets, output):
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Contains the web_package rule.
+"""
+
+def html_asset_inject(index_html, action_factory, injector, root_dirs, assets, output):
+    """Injects JS and CSS resources into the index.html.
+
+    Args:
+      index_html: the input html file
+      action_factory: TODO
```

Comment:
@alexeagle Can you comment with best descriptions for these?

Reply:
I was trying to use Law of Demeter here. We usually pass the whole ctx object but don't use it - plus the ctx object under an Aspect is different.

The doc here could be `Bazel's actions module from ctx.actions - see https://docs.bazel.build/versions/master/skylark/lib/actions.html`

Reply:
Thanks